js/jquery_validation.js

```diff
@@ -0,0 +1,285 @@
+// jQuery validation
+$(document).ready(function()
+{
+	validation();	
+});
+
+/**
+* To start validation
+*
+* @access public
+* @param  void
+* @return void
+*/
+function validation()
+{
+	$('#sign_up_form').submit(function()
+	{
+		var return_val_name = true;
+		text_elements = $('.text_field');
+		$.each(text_elements, function(index, value)
+		{
+			var a = strict_alphabet($(value))
```

Comment:
What this variable name is?
It should be some proper name. Please change in other places too.